crates/wdk-build/src/lib.rs

```diff
@@ -323,54 +358,51 @@ impl Config {
     pub fn get_include_paths(&self) -> Result<Vec<PathBuf>, ConfigError> {
         // FIXME: consider deprecating in favor of iter
         let mut include_paths = vec![];
-
+    
         let include_directory = self.wdk_content_root.join("Include");
-
+    
         // Add windows sdk include paths
         // Based off of logic from WindowsDriver.KernelMode.props &
         // WindowsDriver.UserMode.props in NI(22H2) WDK
         let sdk_version = utils::get_latest_windows_sdk_version(include_directory.as_path())?;
         let windows_sdk_include_path = include_directory.join(sdk_version);
-
-        let crt_include_path = windows_sdk_include_path.join("km/crt");
-        if !crt_include_path.is_dir() {
+    
+        let km_include_path = windows_sdk_include_path.join("km");
+        if !km_include_path.is_dir() {
             return Err(ConfigError::DirectoryNotFound {
-                directory: crt_include_path.to_string_lossy().into(),
+                directory: km_include_path.to_string_lossy().into(),
             });
         }
         include_paths.push(
-            crt_include_path
+            km_include_path
                 .canonicalize()?
                 .strip_extended_length_path_prefix()?,
         );
 
-        let km_or_um_include_path = windows_sdk_include_path.join(match self.driver_config {
-            DriverConfig::Wdm | DriverConfig::Kmdf(_) => "km",
-            DriverConfig::Umdf(_) => "um",
-        });
-        if !km_or_um_include_path.is_dir() {
+        let kit_shared_include_path = windows_sdk_include_path.join("shared");
+        if !kit_shared_include_path.is_dir() {
             return Err(ConfigError::DirectoryNotFound {
-                directory: km_or_um_include_path.to_string_lossy().into(),
+                directory: kit_shared_include_path.to_string_lossy().into(),
             });
         }
         include_paths.push(
-            km_or_um_include_path
+            kit_shared_include_path
                 .canonicalize()?
                 .strip_extended_length_path_prefix()?,
         );
-
-        let kit_shared_include_path = windows_sdk_include_path.join("shared");
-        if !kit_shared_include_path.is_dir() {
+        
+        let um_include_path = windows_sdk_include_path.join("um");
+        if !um_include_path.is_dir() {
             return Err(ConfigError::DirectoryNotFound {
-                directory: kit_shared_include_path.to_string_lossy().into(),
+                directory: um_include_path.to_string_lossy().into(),
             });
         }
         include_paths.push(
-            kit_shared_include_path
+            um_include_path
                 .canonicalize()?
                 .strip_extended_length_path_prefix()?,
         );
-
+    
```

Comment:
Why are you changing the include path logic? This changes how all the bindings are generated and break it.

The logic here strictly matches what happens in the vcxproj files of the WDK. If RC compilation requires different includes, it should be handled elsewhere